build-conf/zCEE3.properties

```diff
@@ -0,0 +1,25 @@
+# Releng properties used by language/zCEE3.groovy
+
+#
+# Comma separated list of required build properties for zCEE3.groovy
+zCEE_requiredBuildProperties=zCEE_alwaysRebuildWAR, zCEE_shellEnvironment,\
+                                zCEE_gradlePath, zCEE_gradle_JAVA_OPTS
+
+
+#
+# When set to true, always rebuild the WAR file, even if one is found in the build folder of the z/OS Connect project folder.
+# When set to false, will pick the existing WAR file if found or will build it if no WAR file is found
+zCEE_alwaysRebuildWAR=true
+
+#
+# Shell environment used to run the gradle command
+zCEE_shellEnvironment=bash
+
+#
+# Path to gradle executable
+zCEE_gradlePath=/var/gradle/gradle-7.6/bin/gradle
```

Comment:
Let's add this into the description of the property as a sample, but keep the definition clean.

Reply:
Done